modules/openshift-argocd-concept.adoc

```diff
@@ -0,0 +1,12 @@
+[id="openshift-argocd-concept"]
+= OpenShift ArgoCD
```

Comment:
Do we really have a OpenShift ArgoCD? If yes, rename to About OpenShift ArgoCD
If not remove this module and move its content to the Overview of OpenShift GitOps section.